easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-1.2.1335-fix_env.patch

```diff
@@ -0,0 +1,16 @@
+commit https://github.com/ComputeCanada/easybuild-easyconfigs/commit/7c6171057562a68086bcb81362f1e997ba4aaeba
+Author: @cmd-ntrf
+Date:   May 4 2019
+
+
```

Comment:
I would add a comment but I don't know what this patch actually does, I just know they CC uses it.

Reply:
Maybe @bartoldeman or @mboisson can provide some more information?

Reply:
Without this patch, RStudio does not inherits environment variables set before launching it. Since R is provided by a separate module, it means RStudio has no idea where R is nor any other library or executables that were loaded before launching RStudio.

Reply:
Thanks, I've included that explanation in https://github.com/easybuilders/easybuild-easyconfigs/pull/12544/commits/a335b99d2dec1fbbf050cb24d2fd3300399640b7